test-utils/src/mollusk.rs

```diff
@@ -123,21 +123,52 @@ pub fn mollusk_add_so_files(
 pub fn mollusk_exec(
     svm: &Mollusk,
     ix: &Instruction,
-    onchain_state: &HashMap<Pubkey, Account>,
-) -> (Vec<PkAccountTup>, InstructionResult) {
+    onchain_state: &AccountMap,
+) -> (AccountMap, InstructionResult) {
```

Comment:
heads up created https://github.com/igneous-labs/inf-1.5/issues/130 to address in another PR